src/pe/mod.rs

```diff
@@ -253,16 +253,18 @@ impl<'a> PE<'a> {
                 }
             }
 
-            if let Some(&baserelocs_dir) =
-                optional_header.data_directories.get_base_relocation_table()
-            {
-                relocation_data = Some(relocation::RelocationData::parse_with_opts(
-                    bytes,
-                    baserelocs_dir,
-                    &sections,
-                    file_alignment,
-                    opts,
-                )?);
+            if opts.parse_basereloc {
+                if let Some(&baserelocs_dir) =
```

Comment:
It would be maybe more idiomatic to write something like:

```rust
relocation_data = get_base_relocation_data().map(|dir|
RelocationData::parse(bytes, dir, etc))
```
 Sorry for not complete example, on phone. But I think you get the gist of using map here and assigning to relocation data so the Some wraps disappear.